indico/modules/rb/models/rooms.py

```diff
@@ -693,14 +693,14 @@ def get_permissions_for_user(cls, user):
                                               'is_reservable'),
                                     raiseload('owner'),
                                     joinedload('acl_entries')))
-        is_admin = cls.is_user_admin(user)
+        is_admin = allow_admin and cls.is_user_admin(user)
```

Comment:
Maybe move it somewhere before the `for room in all_rooms_query` loop since it is the first place where `is_admin` is used.